src/config.c

```diff
@@ -39,7 +39,7 @@ int ReadConfig(RedisModuleString **argv, int argc) {
         printf("loaded default retention policy: %lld \n", TSGlobalConfig.retentionPolicy);
         TSGlobalConfig.hasGlobalConfig = TRUE;
     } else {
-        TSGlobalConfig.retentionPolicy = RETENTION_DEFAULT_SECS;
+        TSGlobalConfig.retentionPolicy = RETENTION_DEFAULT;
```

Comment:
how about `DEFAULT_RETENTION_TIME`

Reply:
:+1: renamed to RETENTION_TIME_DEFAULT like other defaults